devicetypes/sensative/sensative-strips-guard-700.src/sensative-strips-guard-700.groovy

```diff
@@ -0,0 +1,482 @@
+		Integer storedVal = getParamStoredValue(it.num)
+		Integer settingVal = getSettingValue(it.num)
+		
+		if ((settingVal != null) && (settingVal != storedVal)) {
+			logDebug "CHANGING ${it.name}(#${it.num}) from ${storedVal} to ${settingVal}"
+			cmds << configSetCmd(it, settingVal)
+			cmds << configGetCmd(it)
+		}
+		else if (state.pendingRefresh) {
+			cmds << configGetCmd(it)
+		}
+	}
+
+	state.pendingRefresh = false
+	return cmds
+}
+
+
+// Required for HealthCheck Capability, but doesn't actually do anything because this device sleeps.
+def ping() {
+	logDebug "ping()"
+}
+
+
+def refresh() {
+	logDebug "refresh()..."
+
+	state.pendingRefresh = true
+	logForceWakeupMessage("The device will be refreshed the next time it wakes up.")	
+}
+
+void logForceWakeupMessage(String msg) {
+	log.warn "${msg}  To force the device to wake up immediately, move the magnet towards the round end 3 times."
+}
+
+
+String wakeUpIntervalGetCmd() {
+	return secureCmd(zwave.wakeUpV2.wakeUpIntervalGet())
+}
+
+String wakeUpIntervalSetCmd(seconds) {
+	return secureCmd(zwave.wakeUpV2.wakeUpIntervalSet(seconds:seconds, nodeid:zwaveHubNodeId))
+}
+
+String wakeUpNoMoreInfoCmd() {
+	return secureCmd(zwave.wakeUpV2.wakeUpNoMoreInformation())
+}
+
+String batteryGetCmd() {
+	return secureCmd(zwave.batteryV1.batteryGet())
+}
+
+String versionGetCmd() {
+	return secureCmd(zwave.versionV1.versionGet())
+}
+
+String sensorBinaryGetCmd() {
+	return secureCmd(zwave.sensorBinaryV1.sensorBinaryGet())
+}
+
+String configSetCmd(Map param, int value) {
+	return secureCmd(zwave.configurationV2.configurationSet(parameterNumber: param.num, size: param.size, scaledConfigurationValue: value))
+}
+
+String configGetCmd(Map param) {
+	return secureCmd(zwave.configurationV2.configurationGet(parameterNumber: param.num))
+}
+
+String secureCmd(cmd) {
+	try {
+		if (zwaveInfo?.zw?.contains("s") || ("0x98" in device?.rawDescription?.split(" "))) {
+			return zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+		}
+		else {
+			return cmd.format()
+		}
+	}
+	catch (ex) {
+		return cmd.format()
+	}
+}
+
+void sendCommands(List<String> cmds, Integer delay=100) {
+	if (cmds) {
+		def actions = []
+		cmds.each {
+			actions << new physicalgraph.device.HubAction(it)
+		}
+		sendHubCommand(actions, delay)
+	}
+}
+
+
+def parse(String description) {
+	def cmd = zwave.parse(description, commandClassVersions)
+	if (cmd) {
+		zwaveEvent(cmd)
+	}
+	else {
+		log.warn "Unable to parse: $description"
+	}
+
+	updateLastCheckIn()
+	return []
+}
+
+void updateLastCheckIn() {
+	if (!isDuplicateCommand(state.lastCheckInTime, 60000)) {
+		state.lastCheckInTime = new Date().time
+
+		sendEvent(name: "lastCheckIn", value: convertToLocalTimeString(new Date()), displayed: false)
+	}
+}
+
+String convertToLocalTimeString(dt) {
+	try {
+		def timeZoneId = location?.timeZone?.ID
+		if (timeZoneId) {
+			return dt.format("MM/dd/yyyy hh:mm:ss a", TimeZone.getTimeZone(timeZoneId))
+		}
+		else {
```

Comment:
Please remove this line break before the "else" statement.

Reply:
Removed